### PR TITLE
Add profile pic upload under about me section

### PR DESCRIPTION
**Pull request by Jay-kew**
Trying a simple implementation of Blobstore to use for profile pic upload as the messages already have the markdown editor. After uploading, you will be shown the picture but is not saved yet, will be working out the logic to save to datastore and displaying on the user page.

May ignore HomeServlet.java and imagelearn.html for now as they are purely for my understanding and reference while I continue the datastore part

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+//    }
+//
+//    User userData = datastore.getUser(user);
+//
+//    if(userData == null || userData.getProfilePic() == null) {
+//      return;
+//    }
+//
+//    response.getOutputStream().println(userData.getProfilePic());
+//  }
```
Comment (BenMQ) [inline src/main/java/com/google/codeu/servlets/FormHandlerServlet.java]: remove these?

File: src/main/java/com/google/codeu/servlets/FormHandlerServlet.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * When the user submits the form, Blobstore processes the file upload
+ * and then forwards the request to this servlet. This servlet can then
+ * process the request using the file URL we get from Blobstore.
+ */
+@WebServlet("/my-form-handler")
```
Comment (BenMQ) [inline src/main/java/com/google/codeu/servlets/FormHandlerServlet.java]: the path and class names are not super descriptive. 

 ImageFormHandler?

**Review by BenMQ — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/codeu/servlets/HomeServlet.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This servlet prints out the HTML for the homepage. You wouldn't do this in a real codebase,
+ * but this is meant to demonstrate getting a Blobstore URL and using it in a form to allow
+ * a user to upload a file.
+ */
+@WebServlet("/home")
```
Comment (BenMQ) [inline src/main/java/com/google/codeu/servlets/HomeServlet.java:17]: i imagine this is only testing right now, yea? can we move this to something like /upload  or do you plan to delete this servlet at some point?

**Review by BenMQ — APPROVED** [review]

**Comment by Jay-kew** [comment]
Closing this to refer to the most updated [PR#62](https://github.com/team-45-su-19/team-45-codeu/pull/62)

**Pull request closed without merging**